Makefile

```diff
@@ -14,21 +14,21 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 COMMIT_ID = $(shell git describe --abbrev=40 --always --dirty=+ 2>/dev/null)
 GIT_VERSION = $(shell git describe --match='v[0-9]*.[0-9].[0-9]' 2>/dev/null || echo "(unset)")
 
-CONFIG_KUST_DIR:=config/default
-CRD_KUST_DIR:=config/crd
-MGR_KUST_DIR:=config/manager
-KUSTOMIZE_DEFAULT_BASE:=../default
+CONFIG_KUST_DIR := config/default
```

Comment:
I approve of consistency, but I'd prefer it to be consistently without spaces. Would you object to that?